bndtools.core.test/src/simple/pkg/package-info.java

```diff
@@ -0,0 +1,3 @@
+@org.osgi.annotation.bundle.Export
```

Comment:
simple.bnd already has these packages as exported, so we can remove the duplicate Export annotation here.

Reply:
Fair enough, but just curious - what's best practice, to use the annotation or to mark it for export in the .bnd file?